easybuild/tools/module_generator.py

```diff
@@ -132,6 +132,23 @@ def __init__(self, application, fake=False):
         self.log = fancylogger.getLogger(self.__class__.__name__, fname=False)
         self.fake_mod_path = tempfile.mkdtemp()
 
+    def append_paths(self, key, paths, allow_abs=False, expand_relpaths=True):
+        """
+        Generate append-path statements for the given list of paths.
+
+        :param key: environment variable to append paths to
+        :param paths: list of paths to append
+        :param allow_abs: allow providing of absolute paths
+        :param expand_relpaths: expand relative paths into absolute paths (by prefixing install dir)
+        """
+        try:
+            txt = self.prepend_paths(key, paths, allow_abs=allow_abs, expand_relpaths=expand_relpaths)
+        except EasyBuildError as err:
+            raise EasyBuildError(str(err).replace('prepend', 'append'))
+
+        regex = re.compile('^prepend([-_]path)', re.M)
+        return regex.sub(r'append\1', txt)
```

Comment:
I don't really like this.

Reply:
The alternative is to copy-paste the code for `prepend_paths`, and replace `prepend` with `append`. I'm not a big fan of that approach...

Reply:
It would be a less confusing approach though. This one is slightly ugly in my opinion.